recipes/qnm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qnm" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 59d364dacbe5ea7b2b62844aad40f9351d987fe6e3c9c9b0d076574a8033d03f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
I think you just need ``python`` and ``pip`` in ``host``, am I right?

Reply:
Maybe this question belonged here instead. Do I want `--no-deps --ignore-installed` on the pip command line?

Reply:
I've cut down the `host` section to just `pip` and `python`, and added `--no-deps --ignore-installed` to the pip command line